src/wfs/src/main/java/org/geoserver/wfs/FeatureSizeFeatureCollection.java

```diff
@@ -30,8 +30,8 @@
  */
 public class FeatureSizeFeatureCollection extends DecoratingSimpleFeatureCollection {
 
-    /** The default feature cache size */
-    public static final int DEFAULT_CACHE_SIZE = 16;
+    /** The default feature cache size - disabled by default */
+    public static final int DEFAULT_CACHE_SIZE = 0;
```

Comment:
So this results in a change of behaviour, do we need a documentation note saying how to enable the behaviour?

Reply:
I couldn't find any documentation indicating the original change in behavior... but I'm happy to add some. Do you have a recommended spot it should go?

Reply:
maybe https://github.com/geoserver/geoserver/blob/master/doc/en/user/source/production/config.rst ?

Reply:
Either on the WFS page or on the store page?

Actually there is a cache your data heading here - http://docs.geoserver.org/latest/en/user/production/config.html